docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/10-advanced-with-helm.md

```diff
@@ -1,9 +1,38 @@
 ---
 sidebar_label: With Helm
+description: Learn how to perform an advanced installation of Kargo using Helm
 ---
 
 # Advanced Installation with Helm
 
-:::warning
-Placeholder
-:::
+For a customized and secure Kargo setup, follow these advanced installation steps.
```

Comment:
We can't really say it's secure... that's going to depend entirely on how they configure it.

I would just reword this to say something about these being generalized instructions for installing Kargo with non-default configuration.